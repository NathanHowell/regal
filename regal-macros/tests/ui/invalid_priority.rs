use regal_macros::RegalLexer;

#[derive(RegalLexer)]
enum BadPriority {
    #[regex(r"[a-z]+")]
    Word,
    #[token("=")]
    #[priority("high")]
    Eq,
}

fn main() {}
