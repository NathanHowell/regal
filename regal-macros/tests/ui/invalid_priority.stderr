error: priority attribute expects #[priority = <int>]
 --> tests/ui/invalid_priority.rs:8:5
  |
8 |     #[priority("high")]
  |     ^
